app/services/user_validation_service.rb

```diff
@@ -74,10 +74,17 @@ def missing_user_features(db_user)
   def validate_user_handle_no_records
     ValidateResult.new(:pass, nil, url_for(
                                      :controller    => "ems_infra",
-                                     :action        => 'show_list',
-                                     :flash_warning => true,
-                                     :flash_msg     => _("Non-admin users can not access the system until at least 1 VM/Instance has been discovered"))
-                      )
+                                     :action        => 'show_list'))
+  end
+
+  def missing_user_features(db_user)
```

Comment:
Looks like you forgot to remove this method which is also defined a little further up in the file.